packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -122,6 +122,30 @@ abstract class FlutterCommand extends Command<Null> {
     _usesPubOption = true;
   }
 
+  /// Adds flags for using a specific filesystem root and scheme.
+  ///
+  /// Only usable in Dart 2 mode.
```

Comment:
This is redundant now as there is(or very shortly will be) only dart2 mode.

Reply:
I've pulled that comment out.